ts-rs/src/lib.rs

```diff
@@ -407,22 +407,22 @@ pub trait TS {
         deps
     }
 
-    /// Manually export this type to the filesystem. 
```

Comment:
The last commit seems to have removed trailing newlines. That's a problem, I think.

Reply:
It seems to have trimmed off lines with a single trailing whitespace, that shouldn't affect newlines, as they require two trailing whitespaces